packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -243,11 +245,16 @@ type SingleFormComponentProps = {
 function SingleForm({ form, appUrl, Page, enrichedWithUserProfileForm }: SingleFormComponentProps) {
   const utils = trpc.useContext();
   const { t } = useLocale();
+  const { data: teamMembers } = form.teamId
+    ? trpc.viewer.teams.listMembers.useQuery({ teamIds: [form.teamId] })
+    : { data: [] };
```

Comment:
If we are querying the team members everytime the form has a `teamId` can we move this to server side and include it in the `formQuery.handler`?

Reply:
Done